torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -303,6 +303,30 @@ def should_exclude_padding_time(match, arg_name):
     if not fetch_fake_tensors(match, (arg_name,))[0].is_contiguous():
         return False
 
+    # TODO - see issue https://githpub.com/pytorch/pytorch/issues/128889
+    # We would only able to completely plan these out if we were only doing
+    # first dimension padding. non-first we would still need a copy
+    # because these outputs are fixed dense.
+    cannot_plan_output = [
+        aten.mm.default,
```

Comment:
This is just because we force the layout with max-autotune today, right?

Reply:
Yea, i can exclude this when not in max-autotune. Although we'd still need to exclude when padding non first dim

Reply:
eh, actually, i think it'd be better to be consistent..